prow/spyglass/README.md

```diff
@@ -145,11 +145,11 @@ more details and examples.
 Spyglass is currently disabled by default. To enable it, add the `--spyglass` arg to your
 [deck deployment](https://github.com/kubernetes/test-infra/blob/e9e544733854d54403aa1dfd84ca009fd9b942f0/prow/cluster/starter.yaml#L236).
 
-Spyglass config takes the following form:
+To provide any use, Spyglass must by configured. Its config takes the following form:
 ```yaml
 deck:
   spyglass:
-    size_limit: 500e6
+    size_limit: 500000000 # 500MB
```

Comment:
This is probably best written as `500e+6` (as distinct from `500e6`, which is what it currently says).

Reply:
updated